src/locales/en/common.json

```diff
@@ -1937,6 +1937,7 @@
       },
       "history": {
         "tab": "History",
+        "disclaimer": "Your swap desktop transactions are not synchronized with Ledger Live Desktop",
```

Comment:
You probably mean "mobile" at least once :smiley: but not sure which one.

Reply:
Hahahaha indeed, nice catch 🤦🏼